StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet.swift

```diff
@@ -169,7 +169,8 @@ public class PaymentSheet {
                                 configuration: self.configuration,
                                 loadResult: loadResult,
                                 isFlowController: false,
-                                analyticsHelper: self.analyticsHelper
+                                analyticsHelper: self.analyticsHelper,
+                                walletButtonsShownExternally: false
```

Comment:
I think we should either:
A. Remove passing in false since it defaults to false anyways
or
b. Remove the default in the init and leave this as `false`.

Reply:
Makes sense!